qemu/tests/migration_multi_host_with_kdump.py

```diff
@@ -0,0 +1,185 @@
+import logging
+from autotest.client.shared import error
+from autotest.client.shared.syncdata import SyncData
+from virttest import env_process
+from virttest import utils_test
+from generic.tests import kdump
+
+
+@error.context_aware
+def run(test, params, env):
+
+    """
+    KVM multi-host migration ping pong test:
+
+    Migration execution progress is described in documentation
+    for migrate method in class MultihostMigration.
+
+    The test procedure:
+    1) starts vm on master host.
+    2) Configure kdump inside guest
+    3) Implement kdump inside guest and pin the task to 1 cpu
+    4) start ping pong migration master->slave->master variable migrate_count.
+    5) If a migration finished but kdump has not finished,
+       migrate VM back to the source host again
+    6) after kdump, reboot guest
```

Comment:
Yes, I mean, you could use a boolean value to do this, like 'stop_migration = true' instead. My point is, it seems meaningless to set migrate_count = 3 in the cfg file, since it only set to 0 no matter what migrate_count equals to. Thanks.